using-directus-as-a-baby-health-tracker/index.md

```diff
@@ -0,0 +1,362 @@
+  "oxta": 255,
+  "onm": 0,
+  "bsb": 4,
+  "mrs": 0,
+  "hw": "obl"
+}
+```
+
+Some of the keys are self-speaking while others are not clear. Due to the lack of documentation, I can only compare the values with the data from the original app to identify what is what. Also, the mentioned GitHub projects have some details from other users that were helping to understand the keys here.
+In our case, we are using `ox` as the oxgenlevel, `hr` as heartrate, and `bat` as battery level. Also `st` is used which should be the skin temperature.
+
+##### Saving the data within Directus
+The last step within the flow is to save the gathered data within the collection we've set up already. The payload of the create operation  looks like this:
+
+```Json
+{
+    "raw_payload": "{{$last}}",
+    "oxygen": "{{$last.ox}}",
+    "heart_rate": "{{$last.hr}}",
+    "temperature": "{{$last.st}}",
+    "battery_level": "{{$last.bat}}"
+}
+```
+
+From here, we have all the basic data saved within Directus, if the Flow runs without any issues, don't forget to turn off the logging of the flow and maybe also the logging for the data collection to reduce the amount of overhead data within the Flow log and activity table for the collection.
+## Reporting
+### Insights
+As the Insights module is built in Directus, you can create any report with the data just as you need it. In our case, we've created the following dashboard:
+!(Pasted image 20240305062855.png)
+
+The dashboard is split into three sections, on the left, the last 30 minutes of data are displayed. On the right, you can select the amount of hours you'd like to review. This is done by a slide that is used as a global variable for the 3 charts underneath. 
+In between, some stats are displayed, in detail, in the very last dataset for the oxygen level, heart rate, temperature and battery stats. 
+
+All the charts are using the "Time Series" chart. The left side uses the default values for the data range of 30 minutes. For each measurement, the min and max values are set to render the most common values including some border values. The dynamic charts on the right are set up similarly. 
+
+To achieve the dynamic time window, we have to create the "Global Variable" with the name 'last_x_hours' first:
+!(Pasted image 20240223104116.png)
+
+Within the dynamic "Time Series" charts we can reuse the global variable by adding its name to the date range:
+!(Pasted image 20240223104315.png)
+Again we can use the `{{…}}` syntax, followed by a lower `h` to identify the value that should work as hours. If you use the same variable for all three slides, they will change altogether once you've selected a new amount of hours within the global variable slider.
+
+For the basic Metric values, you use the following configuration:
+!(Pasted image 20240223105348.png)
+To make the values more self-speaking, the style and format can be adjusted e.g. by different colours based on the current values.
+!(Pasted image 20240223105507.png)
```

Comment:
Let's remove this as it's more basic insights tutorial stuff

Reply:
removed within a next commit